pkg/apim/HttpMockAPIM.go

```diff
@@ -0,0 +1,73 @@
+package apim
```

Comment:
I believe you can add a comment here for the compiler to not add this to the built binary